code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -593,15 +593,22 @@
 	if(candidates.len)
 		theghost = pick(candidates)
 		var/mob/living/simple_animal/hostile/guardian/G = spawn_guardian(user, theghost.key)
-		var/time_limit = world.time + 300
-		var/guardianNewName = stripped_input(G.summoner, "You are the user of [G.name]. Would you like to name your guardian something else?", "Name Guardian", G.name, MAX_NAME_LEN)
-		if(world.time < time_limit)
-			if(length(guardianNewName) > 0)
+		//Give the stand user 3 chances to rename their stand
+		for(var/i = 2, i >= 0,i--)
```

Comment:
this way it'll play 4 times, till it gets to -1, you probably wanna use > instead of >=